### PR TITLE
[RFR] Add breadcrumb widget

### PR DESCRIPTION
**Pull request by mshriver**
copypasta from ManageIQ/integration_tests/widgetastic_manageiq

### CODE REVIEW

**Comment by mshriver** [comment]
Travis failure is from about modal in py2.7, not related.

File: src/widgetastic_patternfly/__init__.py [hunk 1]
```diff
@@ -1804,3 +1804,48 @@ def items(self):
...
+
+class BreadCrumb(Widget):
+    """ Patternfly BreadCrumb navigation control
+
+    .. code-block:: python
+
+        breadcrumb = BreadCrumb()
+        breadcrumb.click_location(breadcrumb.locations[0])
+    """
+    ROOT = '//ol[@class="breadcrumb"]'
```
Comment (abalakh) [inline src/widgetastic_patternfly/__init__.py]: May i suggest slightly different locator?
`//ol[contains(@class, "breadcrumb")]`?
Breadcrumb in satellite just have 2 classes, so we'll have to override it (which is still ok for us)

Comment (mshriver) [reply src/widgetastic_patternfly/__init__.py]: Double checking MIQ breadcrumbs to know the scope, but sounds like a great improvement.

**Comment by mshriver** [comment]
I don't have merge rights here, yet, @abalakh, we'll have to get @psav to merge I believe.

**Pull request merged**